src/zap_hooks/helpers/utilities.py

```diff
@@ -0,0 +1,71 @@
+from sys import exit
+from typing import Dict, Iterable, NoReturn
+
+from requests import Response
+from requests.exceptions import HTTPError
+
+import src.zap_hooks.helpers.constants as Constants
+from src.zap_hooks.model.log_level import LogLevel, loggerFunc
+from src.zap_hooks.model.target_availability_check import TargetAvailabilityCheck
+
+
+UTF_8: str = 'utf-8'
```

Comment:
can you remove this?

Reply:
yup, removed all unused stuff from the hooks on last commit